documentation/source/usersGuide/usersGuide_12_music21object.ipynb

```diff
@@ -1454,24 +1454,18 @@
  ],
  "metadata": {
   "anaconda-cloud": {},
+  "interpreter": {
+   "hash": "aee8b7b246df8f9039afb4144a1f6fd8d2ca17a180786b69acc140d282b71a49"
+  },
   "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
+   "display_name": "Python 3.10.0b1 64-bit",
```

Comment:
oh -- good to know that music21 is working on 3.10-beta.  I hadn't tested it yet.

Reply:
yeah, the only pain point is some enum repr()s have to be changed in doctests. more later...